java/code/src/com/suse/manager/webui/templates/groups/formula.jade

```diff
@@ -15,8 +15,7 @@
 
 #formula
 
-script(type='text/javascript').
-    var csrfToken = "#{csrf_token}";
+    window.csrfToken = "#{csrf_token}";
```

Comment:
Don't we need the `script(type='text/javascript').` snippet here?

Reply:
Nice catch, I think I messed up while solving some rebase conflicts :sweat:  

I will do a sanity check full review again through the whole PR and then merge it.

Thanks